architecture/core_concepts/routes.adoc

```diff
@@ -621,6 +622,11 @@ the destination. This field is required, but only for re-encryption.
 [[router-sharding]]
 == Router Sharding
```

Comment:
Careful... this isn't the router sharding you talk about at the bottom.  We either need a new name, or we need to talk about both types in the overview and clarify which we mean whenever we say router sharding.